tokio/src/runtime/builder.rs

```diff
@@ -276,7 +277,18 @@ impl Builder {
         })
     }
 
+    // Without rt-full, the "threadpool" variant just uses current-thread
+    #[cfg(not(feature = "rt-full"))]
```

Comment:
Shouldn't the function just not be there?

Reply:
It was easier to do this strategy. The other option would have been to feature gate the build fn. Either way, feature flags will need to be entirely redone.